tests/client_connectivity/test_bridge_mode.py

```diff
@@ -0,0 +1,302 @@
+        staConnect.start()
+        print("napping %f sec" % staConnect.runtime_secs)
+        time.sleep(staConnect.runtime_secs)
+        staConnect.stop()
+        staConnect.cleanup()
+        run_results = staConnect.get_result_list()
+        for result in run_results:
+            print("test result: " + result)
+        # result = 'pass'
+        print("Single Client Connectivity :", staConnect.passes)
+        if staConnect.passes():
+            instantiate_testrail.update_testrail(case_id=TEST_CASES["5g_wpa_bridge"], run_id=instantiate_project,
+                                                 status_id=1,
+                                                 msg='5G WPA Client Connectivity Passed successfully - bridge mode')
+        else:
+            instantiate_testrail.update_testrail(case_id=TEST_CASES["5g_wpa_bridge"], run_id=instantiate_project,
+                                                 status_id=5,
+                                                 msg='5G WPA Client Connectivity Failed - bridge mode')
+        assert staConnect.passes()
+        # C2419
+
+    @pytest.mark.wpa2_personal
+    @pytest.mark.twog
+    def test_client_wpa2_personal_2g(self, request, get_lanforge_data, setup_profile_data, instantiate_project, instantiate_testrail):
+        profile_data = setup_profile_data["BRIDGE"]["WPA2_P"]["2G"]
+        station_names = []
+        for i in range(0, int(request.config.getini("num_stations"))):
+            station_names.append(get_lanforge_data["lanforge_2dot4g_prefix"] + "0" + str(i))
+        staConnect = StaConnect2(get_lanforge_data["lanforge_ip"], int(get_lanforge_data["lanforge-port-number"]),
+                                 debug_=False)
+        staConnect.sta_mode = 0
+        staConnect.upstream_resource = 1
+        staConnect.upstream_port = get_lanforge_data["lanforge_bridge_port"]
+        staConnect.radio = get_lanforge_data["lanforge_2dot4g"]
+        staConnect.resource = 1
+        staConnect.dut_ssid = profile_data["ssid_name"]
+        staConnect.dut_passwd = profile_data["security_key"]
+        staConnect.dut_security = "wpa2"
+        staConnect.station_names = station_names
+        staConnect.sta_prefix = get_lanforge_data["lanforge_2dot4g_prefix"]
+        staConnect.runtime_secs = 10
+        staConnect.bringup_time_sec = 60
+        staConnect.cleanup_on_exit = True
+        # staConnect.cleanup()
+        staConnect.setup()
+        staConnect.start()
+        print("napping %f sec" % staConnect.runtime_secs)
+        time.sleep(staConnect.runtime_secs)
+        staConnect.stop()
+        staConnect.cleanup()
+        run_results = staConnect.get_result_list()
+        for result in run_results:
+            print("test result: " + result)
+        # result = 'pass'
+        print("Single Client Connectivity :", staConnect.passes)
+        if staConnect.passes():
+            instantiate_testrail.update_testrail(case_id=TEST_CASES["2g_wpa2_bridge"], run_id=instantiate_project,
```

Comment:
Thinking this would call out to the reporting subsystem to either call testrail or allure.